content/momentum/4/mta-sts.md

```diff
@@ -0,0 +1,100 @@
+---
```

Comment:
Kris had raised an issue on slack about the new feature not being searchable.  Let's see if the issue gets resolved just by the virtue of merging the PR into `main`.  If not, we'll have to revisit and seek help if needed.